docs/DeveloperGuide.md

```diff
@@ -160,10 +162,23 @@ The following sequence diagram explains the above steps when the user searches f
 
 ![Overview](DG_Diagrams/RouteMapCommandSeq.png)
 
+###3.3. List All stops (/liststops Feature)
+`/liststops` is the command which prints all bus stops declared in the BusStops enum.
+
+The `ListStopsCommand#executeCommand()` method of ListStopsCommand Class executes the command in the following steps:
```

Comment:
```suggestion
The `ListStopsCommand#executeCommand()` method of ListStopsCommand Class executes the command in the following step:
```

Reply:
I think it is still incomplete?

Reply:
yeah

Reply:
i forgot it updates automatically hahaha